daemon/daemon_unix.go

```diff
@@ -375,6 +390,13 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *containertypes.
 		warnings = append(warnings, "IPv4 forwarding is disabled. Networking will not work.")
 		logrus.Warnf("IPv4 forwarding is disabled. Networking will not work")
 	}
+	if hostConfig.CgroupParent != "" && daemon.usingSystemd() {
+		// CgroupParent for systemd cgroup should be named as "xxx.slice"
+		length := len(hostConfig.CgroupParent)
+		if length <= 6 || hostConfig.CgroupParent[length-6:] != ".slice" {
```

Comment:
For slices, you just need the name of the slice, not the entire path. There is a convention that systemd uses to convert that into a full path when there are dashes in the name. @cyphar What issue did you run into?